apps/backend/db_patches/0168_XpressEmailSetting.sql

```diff
@@ -0,0 +1,11 @@
+DO
+$$
+BEGIN
+  IF register_patch('0168_XpressEmailSetting.sql', 'Chi Kai Lam', 'Xpress email setting for recipients SAMPLE_SAFETY_EMAIL and TECHNIQUE_SCIENTISTS_EMAIL', '2025-01-13') THEN
```

Comment:
There's still a reference to the removed technique scientists email setting here